build.gradle

```
@@ -39,6 +45,8 @@ ext {
   slf4jVersion = '1.7.36'
   vavrVersion = '0.10.4'
 
+  intellijSnapshotsUrl = "https://www.jetbrains.com/intellij-repository/"
```

Technically, it's
```suggestion
  intellijSnapshotsUrl = "https://www.jetbrains.com/intellij-repository/snapshots"
```

https://www.jetbrains.com/intellij-repository/ returns a 404 😛